chore: consolidate consensus nodes into one page (#1107)
celestiaorg/docs: docs/nodes/consensus-node.mdx

rm -rf ~/.celestia-app/data<br/>
mkdir -p ~/.celestia-app/data<br/>
SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
    egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
    -C ~/.celestia-app/data/<br/>
</code></pre>

</TabItem>
<TabItem value="arabica" label="Arabica">

> Run the following command to quick-sync from a snapshot for `arabica-10`:

Reviewer: [question] is it possible to interpolate the Arabica chain-id here? Maybe something like: `{constants.arabicaChainId}`?

Author: yep! using the inline text component:

```mdx
This is a sentence with Arabica's chain-id: <InlineText constant={constants.arabicaChainId} />.
```

Author: 48ad87f67c300b20a5254866d1eaa27c3838e9a1 great catch too, fixed on the other instance as well

<pre><code>
cd $HOME<br/>
rm -rf ~/.celestia-app/data<br/>
mkdir -p ~/.celestia-app/data<br/>
SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
    egrep -o ">{constants.arabicaChainId}.*tar" | tr -d ">")<br/>
wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
    -C ~/.celestia-app/data/<br/>
</code></pre>

</TabItem>
</Tabs>
```

### Start the consensus node

In order to start your full consensus node, run the following:

```sh
celestia-appd start
```

> Follow the tutorial on
> [setting up Celestia App as a background process with SystemD](./systemd.md).

Reviewer: [optional] do we want to clarify that this is an optional step?

```suggestion
Optional: If you would like celestia-app to run as a background process, you can follow the [SystemD tutorial](./systemd.md).
```

Author: absolutely, thank you

:::tip
Refer to
[the ports section of the celestia-node troubleshooting page](../../nodes/celestia-node-troubleshooting/#ports)
for information on which ports are required to be open on your machine.
:::

## Optional: Setting up a validator

### Setting up a Celestia validator node

Validator nodes allow you to participate in consensus in the Celestia network.

![validator node](../img/nodes/validator.png)

#### Validator hardware requirements

> The following hardware minimum requirements are recommended for running the
> validator node:

Reviewer: ```suggestion
The following hardware minimum requirements are recommended for running a
validator node:
```

Author: d2cea5356b2a6d07a84b0f778ff47e4a99dc00b8

Author: whoops my b i meant to just commit your suggestion

RPC service listen on all available network interfaces.

#### Note on `external-address`

The `external-address` field in the configuration is used when your node
is behind a NAT and you need to advertise a different address for peers
to dial. Populating this field is not necessary for making the RPC
endpoint public.

```sh
EXTERNAL-ADDRESS=$(wget -qO- eth0.me)
sed -i.bak -e "s/^external-address = ""/external-address = "$EXTERNAL-ADDRESS:26656"/" $HOME/.celestia-app/config/config.toml
```

#### Restart the node

After making these changes, restart `celestia-appd` to load the new
configurations.

### Optional: Transaction indexer configuration options

This section will show you how to set your `config.toml` file in `celestia-app`
to chose which transactions to index. In some
cases, a node operator will be able to decide which transactions to index
based on configuration set in the application.

The options are:

1. `null`
2. `kv` (default) - the simplest possible indexer backed by key-value storage
   (defaults to levelDB; see DBBackend)
   1. when `kv` is chosen `tx.height` and `tx.hash` will always be indexed
3. `psql` - the indexer services backed by PostgreSQL

>    1. when `kv` or `psql` is chosen, `tx.height` and `tx.hash` will always be indexed

Reviewer: ```suggestion
   1. when `psql` is chosen, `tx.height` and `tx.hash` will always be indexed
```
